main.yml

```diff
@@ -354,20 +392,21 @@ ingress:
 insights:
   title: Insights
   frontend:
-    title: Red Hat Insights
+    title: Insights
     sub_apps:
       - id: dashboard
         default: true
       - id: advisor
+      - id: drift
+      - id: inventory
       - id: vulnerability
       - id: compliance
-      - id: patch
-      - id: drift
       - id: policies
-      - id: inventory
-      - id: remediations
+      - id: patch
+      - id: subscriptions
+        section: business
       - id: registration
-      - id: ros
```

Comment:
I would like to know - is `ros` sub-app unintentionally removed from qa-stable or any reason behide it?

Reply:
@ryelo probably took what is on `prod-beta` and mapped it to `qa-stable`. Which envs should have the `ros` application?

Reply:
It is not yet in prod-beta but `ros` application was present in CI/QA/Stage.